purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -2376,7 +2392,7 @@ class PurchasesTest {
         val networkUserID = "networkUserID"
         mockAdInfo(true, networkUserID)
 
-        Purchases.addAttributionData(mapOf("key" to "value"), network, networkUserID)
+        Purchases.addAttributionData(mapOf("key" to "value"), Purchases.AttributionNetwork.APPSFLYER, networkUserID)
```

Comment:
isn't this the same as `network`? I'm a bit confused here

Reply:
`network` is a `AttributionNetwork` from the `common` package. This one is the old `Purchases.AttributionNetwork` that's only used in the public api of `Purchases`. I made an import alias to `CommonAttributionNetwork` to make this more clear